packages/modeling/src/operations/modifiers/edges.js

```diff
@@ -0,0 +1,204 @@
+const vec3 = require('../../maths/vec3')
+const line3 = require('../../maths/line3')
+
+const poly3 = require('../../geometries/poly3')
+
+// an edge is an array of two vertices
+const addEdge = (edges, edge, polygon) => {
+  let ei = edges.findIndex((element) => {
+    if (element) {
+      if (vec3.equals(element[0], edge[0]) && vec3.equals(element[1], edge[1])) return true
```

Comment:
These comparisons do not use the JSCAD-wide Epsilon to detect edges that are "close enough".  Should they?

Reply:
Yeah. Edges are expected to be precise, which means 'snap' should be performed beforehand.